### PR TITLE
[BAD] Merge CLI install and authenticate instructions into single location

### PR DESCRIPTION
**Pull request by StackScribe**
Signed-off-by: Meg McRoberts <meg.mcroberts@dynatrace.com>

The docs had duplicate instructions for installing and authenticating the CLI -- one set under "Installation", the other on the introductory page for the CLI reference pages.  This PR:

* Merges the instructions into files under "Installation"
* Replaces the instructions in the CLI intro with links to the instructions in "Installation"

### CODE REVIEW

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md [hunk 1]
```diff
@@ -7,113 +7,123 @@ weight: 60
...
+To authenticate the Keptn CLI against the Keptn cluster,
+the exposed Keptn endpoint and API token are required. 
 After [installing Keptn](../helm-install), you already have your Keptn endpoint.
 
-<details><summary>Get API Token and Authenticate Keptn CLI on **Linux / MacOS**</summary>
-<p>
 
-* Set the environment variable `KEPTN_API_TOKEN`:
+Get the Keptn endpoint from the `api-gateway-nginx`.
+If you are using port-forward to expose Keptn,
```
Comment (mowies) [inline content/docs/1.0.x/install/authenticate-cli-bridge/index.md:21]:
```suggestion
If you are using port-forwarding to expose Keptn,
```

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md [hunk 2]
```diff
@@ -7,113 +7,123 @@ weight: 60
...
+your endpoint is `localhost` and the `port` to which you forwarded Keptn
+such as `http://localhost:8080`.)
```
Comment (mowies) [inline content/docs/1.0.x/install/authenticate-cli-bridge/index.md:23]:
```suggestion
such as `http://localhost:8080`.
```

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md [hunk 3]
```diff
@@ -7,113 +7,123 @@ weight: 60
...
-KEPTN_API_TOKEN=$(kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 --decode)
-```
+   ```
+   kubectl -n keptn get service api-gateway-nginx
+   ```
 
-* To authenticate the CLI against the Keptn cluster, use the [keptn auth](../../reference/cli/commands/keptn_auth) command:
+   ```
+   NAME                TYPE        CLUSTER-IP    EXTERNAL-IP                  PORT(S)   AGE
+   api-gateway-nginx   ClusterIP   10.117.0.20   <ENDPOINT_OF_API_GATEWAY>    80/TCP    44m
```
Comment (mowies) [inline content/docs/1.0.x/install/authenticate-cli-bridge/index.md:31]: The `EXTERNAL_IP` field will only show the endpoint/IP if the service is set to type `LoadBalancer`. In the port forwarding case this field is not  set at all.

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md [hunk 4]
```diff
@@ -7,113 +7,123 @@ weight: 60
...
+       -ojsonpath={.data.keptn-api-token} | base64 --decode)
+   ```
 
-For the Windows PowerShell, a small script is provided that installs the `PSYaml` module and sets the environment variables.
+1. To authenticate the CLI against the Keptn cluster,
+   use the [keptn auth](../../reference/cli/commands/keptn_auth) command:
 
-* Set the environment variable `KEPTN_ENDPOINT`:
+   ```
+   keptn auth --endpoint=$KEPTN_ENDPOINT --api-token=$KEPTN_API_TOKEN
```
Comment (mowies) [inline content/docs/1.0.x/install/authenticate-cli-bridge/index.md:50]: You're using the `KEPTN_ENDPOINT` variable here but you're not showing how it is set before that...

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md [hunk 5]
```diff
@@ -7,113 +7,123 @@ weight: 60
...
-* Decode the file:
+   ```
+   kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token}
+   ```
 
-```
-certutil -decode keptn-api-token-base64.txt keptn-api-token.txt
-```
+   ```
+   abcdefghijkladfaea
```
Comment (mowies) [inline content/docs/1.0.x/install/authenticate-cli-bridge/index.md:101]: this is probably not right :)

File: content/docs/1.0.x/install/cli-install/index.md [hunk 6]
```diff
@@ -26,46 +26,102 @@ You can install the Keptn CLI using:
 
 ## curl
 
-```
-curl -sL https://get.keptn.sh/ | bash
-```
+You can use the `curl` command to install the Keptn CLI on Linux, WSL2, and macOS.
+Windows users need `bash`, `curl`, and `awk` installed (e.g., using Git Bash).
```
Comment (mowies) [inline content/docs/1.0.x/install/cli-install/index.md:30]: we are recommending bash on WSL2 here, but in the other file, we explain how to do things in PowerShell. I think we can get rid of the PowerShell instructions and stick with bash for Windows

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md (hunk 5 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/authenticate-cli-bridge/index.md:101]: Sorry, that is what is in both old versions.  What would be a better string?

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md (hunk 1 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/authenticate-cli-bridge/index.md:21]: done

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md (hunk 2 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/authenticate-cli-bridge/index.md:23]: done

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md (hunk 3 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/authenticate-cli-bridge/index.md:31]: Oh ugh!  This all needs to be rewritten but not today.  So I put a link to the access page for details about how to get the endpoint for each access option and made this a "For example, if you are using LoadBalancer..."  See if this works for now.

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md (hunk 4 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/authenticate-cli-bridge/index.md:50]: Ugh.  I now mention it and link to "Choose access option" for instructions about getting the endpoint, depending on the access option being used, and storing it in an environment variable.  It's ugly but I think it is good enough for now.

File: content/docs/1.0.x/install/cli-install/index.md (hunk 6 above)
Comment (StackScribe) [reply content/docs/1.0.x/install/cli-install/index.md:30]: I removed the PowerShell instructions.  Also added a link to the WSL2 documentation the first time it is mentioned (for installing Keptn CLI with curl)

File: content/docs/1.0.x/install/access/index.md [hunk 7]
```diff
@@ -22,85 +22,103 @@ caption="Installation options"
...
 1. **Install Keptn:** Use the Helm CLI to install Keptn on your cluster.
-If you want to install the execution plane for continuous delivery, add the flag `continuousDelivery.enabled=true`.
-  ```console
-   helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace --set=continuousDelivery.enabled=true,apiGatewayNginx.type=LoadBalancer
-  ```
+   If you want to install the execution plane for continuous delivery,
+   add the flag `continuousDelivery.enabled=true`.
+
+   ```
+   helm install keptn keptn/keptn -n keptn --version=$KeptnVersion create-namespace \
```
Comment (mowies) [inline content/docs/1.0.x/install/access/index.md:33]:
```suggestion
   helm install keptn keptn/keptn -n keptn --version=$KeptnVersion --create-namespace \
```

File: content/docs/1.0.x/install/access/index.md [hunk 8]
```diff
@@ -153,32 +171,32 @@ If you want to install the execution plane for continuous delivery, add the flag
...
-    spec:
-      rules:
-      - host: <IP-ADDRESS>.nip.io
-        http:
-          paths:
-          - backend:
-              serviceName: api-gateway-nginx
-              servicePort: 80
       ```
+      apiVersion: networking.k8s.io/v1beta1
```
Comment (mowies) [inline content/docs/1.0.x/install/access/index.md:181]:
```suggestion
      apiVersion: networking.k8s.io/v1
```

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md [hunk 9]
```diff
@@ -18,9 +18,15 @@ After [installing Keptn](../helm-install), you already have your Keptn endpoint.
...
 
 Get the Keptn endpoint from the `api-gateway-nginx`.
-If you are using port-forward to expose Keptn,
+The specifics for getting the Keptn endpoint vary slightly
+depending on the access option you are using for Keptn.
+Instructions for getting the Keptn endpoint and, optionally,
+storing that endpoint in an environment variable
+are detailed for each option in [Choose access options](../access).
+
+For example, if you are using `LoadBalancer` to expose Keptn,
```
Comment (mowies) [inline content/docs/1.0.x/install/authenticate-cli-bridge/index.md:27]: This paragraph doesn't make sense to me. You're talking about a load balancer endpoint here. But then you're providing localhost and port as they would be in a port forwarding setup. And then after that you show the api-gateway-nginx service which is configured in `ClusterIP` mode and not `LoadBalancer` mode. That service can only have an external IP if it's set to `LoadBalancer` mode. Never in `ClusterIP` mode.

File: content/docs/1.0.x/install/authenticate-cli-bridge/index.md (hunk 5 above)
Comment (mowies) [reply content/docs/1.0.x/install/authenticate-cli-bridge/index.md:100]: you can directly decode the token with `base64`:
```
kubectl get secret keptn-api-token -n keptn -ojsonpath={.data.keptn-api-token} | base64 -d
```

**Comment by StackScribe** [comment]
This PR is screwed up in ways I do not understand!  It is replaced by https://github.com/keptn/keptn.github.io/pull/1562/

**Comment by mowies** [comment]
@StackScribe so i guess we can close it then

**Pull request closed without merging**